src/nameko_rediskn/rediskn.py

```diff
@@ -209,8 +234,6 @@ def _subscribe(self):
     def _kill_thread(self):
         if self._thread is not None:
             self._thread.kill()
-            self._thread = None
```

Comment:
Could you please remind me again why we no longer need to do `self._thread = None` and `self.client = None`?

Reply:
👍